packages/text-editor/src/components/StyledTextEditor.svelte

```diff
@@ -300,16 +315,16 @@
           }
         }
       }
+    }
 
-      &.focusable {
-        margin: -0.25rem -0.5rem;
-        padding: 0.25rem 0.5rem;
-        border: 1px solid transparent;
-        border-radius: 0.25rem;
+    &.focusable {
+      border-width: 0.0625rem;
```

Comment:
Minor: use shorthand `border` style?

Reply:
I want to update only border color here on `focus-within`, that is why I am using it like this

Reply:
It doesn't mean you cannot use the shorthand property outside of `focus-within`

Reply:
Updated